src/jukebox/components/rfid/hardware/generic_nfcpy/generic_nfcpy.py

```diff
@@ -0,0 +1,134 @@
+# Standard imports from python packages
+import logging
+
+import nfc
+import glob
+from nfc.clf import RemoteTarget
+
+# Import the ReaderBaseClass for common API. Leave as this line as it is!
+from components.rfid import ReaderBaseClass
+import jukebox.cfghandler
+import misc.inputminus as pyil
+from misc.simplecolors import Colors
+
+# Also import the description string into this module, to make everything available in a single module w/o code duplication
+# Leave this line as is!
+from .description import DESCRIPTION
+
+# Create logger.
+logger = logging.getLogger('jb.rfid.nfcpy')
+# Get the global handler to the RFID config
+cfg = jukebox.cfghandler.get_handler('rfid')
+
+
+def query_customization() -> dict:
+    # filter all log records from nfc.clf
+    logger = logging.getLogger('nfc.clf')
+    logger.filter = lambda record: 0
```

Comment:
This currently overwrites the jb logger, silencing also wanted messages.
```suggestion
    loggerNfcClf = logging.getLogger('nfc.clf')
    loggerNfcClf.filter = lambda record: 0
```